README.md

```diff
@@ -1,5 +1,5 @@
 ## wg21
- [![Travis Build Status](https://travis-ci.org/BobSteagall/wg21.svg?branch=master)](https://travis-ci.org/BobSteagall/wg21)
+ [![Travis Build Status](https://travis-ci.com/BobSteagall/wg21.svg?branch=master)](https://travis-ci.com/BobSteagall/wg21)
```

Comment:
Migrate to Travis.com as Travis.org has now been decommissioned. @BobSteagall needs to create an account with Travis.com using your github sign in and then ensure the CI environment variables set up previously for Travis.org are set up in the setting for Travis.com.